File: CHPT/source/tags/V0.4.7/client_app/Model/tb_organization.cs
using System;
namespace Model
{
	/// <summary>
	/// 组织信息
	/// </summary>
	[Serializable]
	public partial class tb_organization
	{
		public tb_organization()
		{}
		#region Model
		private string _org_id;
		private string _com_id;
		private string _org_code;
		private string _org_name;
		private string _org_short_name;
		private string _parent_id;
		private string _contact_name;
		private string _contact_telephone;
		private string _remark;
		private string _data_sources;
		private string _status;
		private string _enable_flag;
		private string _create_by;
		private long _create_time;
		private string _update_by;
		private long? _update_time;
		/// <summary>
		/// 组织id
		/// </summary>
		public string org_id
		{
			set{ _org_id=value;}
			get{return _org_id;}
		}
		/// <summary>
		/// 所属公司，关联公司档案表
		/// </summary>
		public string com_id
		{
			set{ _com_id=value;}
			get{return _com_id;}
		}
		/// <summary>
		/// 组织编码
		/// </summary>
		public string org_code
		{
			set{ _org_code=value;}
			get{return _org_code;}
		}
		/// <summary>
		/// 组织名称
		/// </summary>
		public string org_name
		{
			set{ _org_name=value;}
			get{return _org_name;}
		}
		/// <summary>
		/// 组织简称
		/// </summary>
		public string org_short_name
		{
			set{ _org_short_name=value;}
			get{return _org_short_name;}
		}
		/// <summary>
		/// 父级，关联自己
		/// </summary>
		public string parent_id
		{
			set{ _parent_id=value;}
			get{return _parent_id;}
		}
		/// <summary>
		/// 联系人
		/// </summary>
		public string contact_name
		{
			set{ _contact_name=value;}
			get{return _contact_name;}
		}
		/// <summary>
		/// 联系电话
		/// </summary>
		public string contact_telephone
		{
			set{ _contact_telephone=value;}
			get{return _contact_telephone;}
		}
		/// <summary>
		/// 备注
		/// </summary>
		public string remark
		{
			set{ _remark=value;}
			get{return _remark;}
		}
		/// <summary>
		/// 数据来源，关联字典码表
		/// </summary>
		public string data_sources
		{
			set{ _data_sources=value;}
			get{return _data_sources;}
		}
		/// <summary>
		/// 状态
		/// </summary>
		public string status
		{
			set{ _status=value;}
			get{return _status;}
		}
		/// <summary>
		/// 删除标记，0删除，1未删除
		/// </summary>
		public string enable_flag
		{
			set{ _enable_flag=value;}
			get{return _enable_flag;}
		}
		/// <summary>
		/// 创建人，关联人员表
		/// </summary>
		public string create_by
		{
			set{ _create_by=value;}
			get{return _create_by;}
		}
		/// <summary>
		/// 创建时间
		/// </summary>
		public long create_time
		{
			set{ _create_time=value;}
			get{return _create_time;}
		}
		/// <summary>
		/// 最后编辑人，关联人员表
		/// </summary>
		public string update_by
		{
			set{ _update_by=value;}
			get{return _update_by;}
		}
		/// <summary>
		/// 最后编辑时间
		/// </summary>
		public long? update_time
		{
			set{ _update_time=value;}
			get{return _update_time;}
		}
		#endregion Model

	}
}

File: CHPT/source/trunk/supp_app/SspDispatchService/src/com/ctfo/storage/model/permission/TbOrganization.java
package com.ctfo.storage.model.permission;

import java.io.Serializable;

/**
 * 
 * 
 * <p>
 * ----------------------------------------------------------------------------- <br>
 * 工程名 ： SspDispatchService <br>
 * 功能： 组织信息<br>
 * 描述： 组织信息<br>
 * 授权 : (C) Copyright (c) 2011 <br>
 * 公司 : 北京中交慧联信息科技有限公司 <br>
 * ----------------------------------------------------------------------------- <br>
 * 修改历史 <br>
 * <table width="432" border="1">
 * <tr>
 * <td>版本</td>
 * <td>时间</td>
 * <td>作者</td>
 * <td>改变</td>
 * </tr>
 * <tr>
 * <td>1.0</td>
 * <td>2014-11-4</td>
 * <td>xuehui</td>
 * <td>创建</td>
 * </tr>
 * </table>
 * <br>
 * <font color="#FF0000">注意: 本内容仅限于[北京中交慧联信息科技有限公司]内部使用，禁止转发</font> <br>
 * 
 * @version 1.0
 * 
 * @author xuehui
 * @since JDK1.6
 */
public class TbOrganization implements Serializable {

	/** */
	private static final long serialVersionUID = -6490074912071157700L;

	/** 组织id */
	private String orgId;

	/** 所属公司，关联公司档案表 */
	private String comId;

	/** 组织编码 */
	private String orgCode;

	/** 组织名称 */
	private String orgName;

	/** 组织简称 */
	private String orgShortName;

	/** 父级，关联自己 */
	private String parentId;

	/** 联系人 */
	private String contactName;

	/** 联系电话 */
	private String contactTelephone;

	/** 备注 */
	private String remark;

	/** 数据来源，关联字典码表 */
	private String dataSources;

	/** 状态 */
	private String status;

	/** 删除标记，0删除，1未删除 */
	private String enableFlag;

	/** 创建人，关联人员表 */
	private String createBy;

	/** 创建时间 */
	private Long createTime;

	/** 最后编辑人，关联人员表 */
	private String updateBy;

	/** 最后编辑时间 */
	private Long updateTime;

	/** 服务站id，云平台用 */
	private String serStationId;

	/** 帐套id，云平台用 */
	private String setBookId;

	public String getSerStationId() {
		return serStationId;
	}

	public void setSerStationId(String serStationId) {
		this.serStationId = serStationId;
	}

	public String getSetBookId() {
		return setBookId;
	}

	public void setSetBookId(String setBookId) {
		this.setBookId = setBookId;
	}

	public String getOrgId() {
		return orgId;
	}

	public void setOrgId(String orgId) {
		this.orgId = orgId;
	}

	public String getComId() {
		return comId;
	}

	public void setComId(String comId) {
		this.comId = comId;
	}

	public String getOrgCode() {
		return orgCode;
	}

	public void setOrgCode(String orgCode) {
		this.orgCode = orgCode;
	}

	public String getOrgName() {
		return orgName;
	}

	public void setOrgName(String orgName) {
		this.orgName = orgName;
	}

	public String getOrgShortName() {
		return orgShortName;
	}

	public void setOrgShortName(String orgShortName) {
		this.orgShortName = orgShortName;
	}

	public String getParentId() {
		return parentId;
	}

	public void setParentId(String parentId) {
		this.parentId = parentId;
	}

	public String getContactName() {
		return contactName;
	}

	public void setContactName(String contactName) {
		this.contactName = contactName;
	}

	public String getContactTelephone() {
		return contactTelephone;
	}

	public void setContactTelephone(String contactTelephone) {
		this.contactTelephone = contactTelephone;
	}

	public String getRemark() {
		return remark;
	}

	public void setRemark(String remark) {
		this.remark = remark;
	}

	public String getDataSources() {
		return dataSources;
	}

	public void setDataSources(String dataSources) {
		this.dataSources = dataSources;
	}

	public String getStatus() {
		return status;
	}

	public void setStatus(String status) {
		this.status = status;
	}

	public String getEnableFlag() {
		return enableFlag;
	}

	public void setEnableFlag(String enableFlag) {
		this.enableFlag = enableFlag;
	}

	public String getCreateBy() {
		return createBy;
	}

	public void setCreateBy(String createBy) {
		this.createBy = createBy;
	}

	public Long getCreateTime() {
		return createTime;
	}

	public void setCreateTime(Long createTime) {
		this.createTime = createTime;
	}

	public String getUpdateBy() {
		return updateBy;
	}

	public void setUpdateBy(String updateBy) {
		this.updateBy = updateBy;
	}

	public Long getUpdateTime() {
		return updateTime;
	}

	public void setUpdateTime(Long updateTime) {
		this.updateTime = updateTime;
	}
}